examples/angularjs-perf/index.html

```diff
@@ -8,45 +8,51 @@
 		<style>[ng-cloak] { display: none; }</style>
 	</head>
 	<body>
-		<section id="todoapp" ng-controller="TodoCtrl">
+		<section id="todoapp" ng-controller="TodoCtrl as TC">
```

Comment:
Hi Igor, cheers for taking time to comment, really appreciated!

I see `TC` as alias/abbreviation for `TodoCtrl`. It does indeed look unusual. 
But also I feel, its nature is different from any other variable in the template.
It is not a property of the controller but rather the controller itself.

*Pros:*
- Instantly recognisable, does not blend with the rest, makes the markup more readable.
- Angular adds to the already heavy HTML markup, so any readable distinction helps.
- Mirrors the other important (and unusual) "Controller as" syntax.

Other possible names:

CamelCase `todoCtrl`:

*Cons:*
- `ng-controller="TodoCtrl as todoCtrl"` feels a bit long and awkward and raises the question about possible misprints, as the two names look too similar. 
- Problems with too similar names: too easy to break the code with tiny misprints, just forgetting to press Shift can lead to unpredictable behaviour.
- The longer name adds to the heavy HTML markup and blends with the rest of the variables.

*Pros:*
- Looks like regular variable, which also makes it harder to recognise.

---
All lowercase `tc`:

*Pros:*
- `TodoCtrl as tc` reads better.

*Cons*
- `tc.addTodo` blends with different variables e.g. `todo.title`. Here `tc` is the Controller alias, where `todo` is a local variable on the `$scope`. However, they look exactly the same, making the code harder to read.
- The same problem inside the controller. The alias `var tc = this` looks exactly the same as any other local variable, yet its role is different and unique. Again, the name makes it blend rather than stand out.

---
Hopefully, this explains the motivation behind, it really is a difficult choice. If the community still thinks that should be changed, I'll be happy to do so.